NGitLab/Models/Position.cs

```diff
@@ -0,0 +1,21 @@
+using System.Text.Json.Serialization;
+
+namespace NGitLab.Models;
+
+public class Position
+{
+    [JsonPropertyName("new_path")]
+    public string NewPath { get; set; }
+
+    [JsonPropertyName("old_line")]
+    public int OldLine { get; set; }
+
+    [JsonPropertyName("new_line")]
+    public int NewLine { get; set; }
+
+    [JsonPropertyName("position_type")]
+    public string PositionType { get; set; }
```

Comment:
Can you introduce an enum for this? Currently allowed values seem to be `text` or `image`, but to make this more robust and support any other eventual types. you could wrap the enum into a `DynamicEnum<TEnum>`.